std/file.d

```diff
@@ -2318,7 +2319,7 @@ private bool ensureDirExists(in char[] pathname)
  * Throws: $(D FileException) on error.
  */
 
-void mkdirRecurse(in char[] pathname)
+void mkdirRecurse()(in char[] pathname)
```

Comment:
The first thing that comes to mind is that you can't take its address anymore. When it's not a template, the function name can be used on its own, but once it is, it has to be instantiated. When calling it, that's not a big deal, because IFTI takes care of the instantiation for you, but for other cases, it has to be done manually.

Of course, even changing a single attribute is an API change, so we're actually really terrible in general about maintaining API compatibility. We usually do a fairly good job with avoiding breaking the common cases, but we frequently break the corner cases.